config/locales/views/settings/api_keys/hu.yml

```
@@ -24,53 +31,86 @@ hu:
+        created_label: "Létrehozva:"
+        security_note_title: "Fontos biztonsági megjegyzés"
+        security_note_body: "Ez az egyetlen alkalom, amikor az API-kulcsod megjelenik. Mindenképpen másold le most és tárold biztonságosan. Ha elveszíted, újat kell generálnod."
+        usage_instructions_title: "Az API-kulcs használata"
         key_name: "Név"
         permissions: "Jogosultságok"
         critical_warning_title: "⚠️ Kritikus: Mentsd el az API-kulcsodat most"
         critical_warning_1: "Ez az egyetlen alkalom, amikor az API-kulcsod egyszerű szövegként látható."
         critical_warning_2: "Másold le és tárold biztonságosan a jelszókezelődben vagy az alkalmazásodban."
         critical_warning_3: "Ha elveszíted ezt a kulcsot, újat kell létrehoznod."
-        usage_instructions_title: "Gyors kezdés"
+        usage_instructions_title: "Az API-kulcs használata"
```

YAMLlint (1.38.0)

[error] duplication of key "usage_instructions_title" in mapping

(key-duplicates)